Source/Core/OpenStreetMapNominatimGeocoderService.js

```diff
@@ -0,0 +1,59 @@
+/*global define*/
+define([
+    './Cartesian3',
+    './defaultValue',
+    './loadJson',
+    './Rectangle'
+], function(
+    Cartesian3,
+    defaultValue,
+    loadJson,
+    Rectangle) {
+    'use strict';
+
+    /**
+     * Provides geocoding through OpenStreetMap Nominatim.
+     * @alias OpenStreetMapNominatimGeocoder
+     *
+     */
+    function OpenStreetMapNominatimGeocoder(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+        this.displayName = defaultValue(options.displayName, 'Nominatim');
+        this._canceled = false;
+        this.autoComplete = defaultValue(options.autoComplete, true);
+    }
+
+    OpenStreetMapNominatimGeocoder.prototype.cancel = function() {
+        this._canceled = true;
+    };
+
+    /**
+     * @function
```

Comment:
You probably don't need `@function` here?  At least I don't think we use this in most other places